src/app/shared/ui/modal/modal.component.html

```
@@ -1,12 +1,12 @@
-<div class="backdrop" (click)="backdropClick.emit()"></div>
+<div class="backdrop" tabindex="0" (keypress)="backdropClick.emit()" (click)="backdropClick.emit()"></div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)